Scripts/runcts

```diff
@@ -84,7 +84,7 @@ fi
 # -------------- MoltenVK configuration --------------------
 
 # As documented above, the portability option restricts to Vulkan 1.0 and a very limited set of extensions.
-# The values used here are documented in mvk_config.h.
+# The values used here are documented in MoltenVK_Configuration_Parameters.md.
 # - MVK_CONFIG_API_VERSION_TO_ADVERTISE = VK_API_VERSION_1_0 (4194304)
 # - MVK_CONFIG_ADVERTISE_EXTENSIONS selects support for a very limited set of extensions,
 #   using a bit-or of values in MVKConfigAdvertiseExtensions (extension list documented above).
```

Comment:
Since you're here, line 93 should probably be updated.

Reply:
Thanks. Good catch. Fixed.